### PR TITLE
DO NOT MERGE feat(Docs): Added redirects for landing and auto-index pages DOC-7617

### PR DESCRIPTION
**Pull request by paperclypse**
All the work from my side is done.

### CODE REVIEW

**Comment by paperclypse** [comment]
This PR is related to #4226

File: src/nav/root.yml [hunk 1]
```diff
@@ -3,57 +3,57 @@ path: /
...
   - title: Distributed tracing
     path: /docs/distributed-tracing
   - title: Errors inbox
     path: /docs/errors-inbox/errors-inbox
   - title: Infrastructure monitoring
-    path: /docs/infrastructure
+    path: /docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring/
   - title: Kubernetes and Pixie
-    path: /docs/kubernetes-pixie
+    path: /docs/kubernetes-pixie/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration
```
Comment (rhetoric101) [inline src/nav/root.yml:27]: @paperclypse, this page generates a 404 for me locally.

File: src/nav/root.yml [hunk 2]
```diff
@@ -3,57 +3,57 @@ path: /
...
   - title: Glossary
     path: /docs/glossary/glossary
   - title: 'Release notes'
     path: '/docs/release-notes'
   - title: Security and privacy
-    path: /docs/security
+    path: /docs/security/overview
   - title: Licenses
-    path: /docs/licenses
+    path: /docs/licenses/overview
```
Comment (rhetoric101) [inline src/nav/root.yml:56]: @paperclypse, you have several "overview.mdx" files at the parent nodes (the same level as landing pages). I thought we were moving away to the model where the default page for every section is the first child below the top node.

File: src/data/taxonomy-redirects.json [hunk 3]
```diff
@@ -361,12 +367,13 @@
     ]
   },
   {
-    "url": "/docs/alerts-applied-intelligence",
+    "url": "/docs/alerts-applied-intelligence/overview",
```
Comment (rhetoric101) [inline src/data/taxonomy-redirects.json:370]: @paperclypse, this overview topic is at a parent level and not a child of the first empty node. Aren't we trying for a model where the initial page is always the child of an empty node?

File: src/data/taxonomy-redirects.json [hunk 4]
```diff
@@ -600,7 +608,7 @@
     ]
   },
   {
-    "url": "/docs/apm/agents",
+    "url": "https://developer.newrelic.com/instant-observability/?search=&category=application-monitoring",
```
Comment (rhetoric101) [inline src/data/taxonomy-redirects.json:611]: @paperclypse , Why are we sending users to the developer site?

File: src/data/taxonomy-redirects.json [hunk 5]
```diff
@@ -832,14 +840,15 @@
     ]
   },
   {
-    "url": "/docs/infrastructure/install-infrastructure-agent/linux-installation",
+    "url": "https://developer.newrelic.com/instant-observability/?search=linux",
```
Comment (rhetoric101) [inline src/data/taxonomy-redirects.json:843]: @paperclypse, like above, why are we sending users to the developer site about Linux?

File: src/data/taxonomy-redirects.json [hunk 6]
```diff
@@ -1255,9 +1266,10 @@
     ]
   },
   {
-    "url": "/docs/apm/agents/java-agent/additional-installation",
+    "url": "https://developer.newrelic.com/instant-observability/?search=java",
```
Comment (rhetoric101) [inline src/data/taxonomy-redirects.json:1269]: @paperclypse, another link to the developer site?

File: src/data/taxonomy-redirects.json [hunk 7]
```diff
@@ -2501,5 +2519,77 @@
     "paths": [
       "/docs/using-new-relic/cross-product-functions/install-configure"
     ]
-  }    
+  },
+  {
+    "url": "/docs/apm/new-relic-apm/getting-started/introduction-apm",
+    "paths": [
+      "/docs/apm"
```
Comment (rhetoric101) [inline src/data/taxonomy-redirects.json:2526]: @paperclypse this URL 404s.

File: src/data/taxonomy-redirects.json [hunk 8]
```diff
@@ -2501,5 +2519,77 @@
...
+  {
+    "url": "/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring",
+    "paths": [
+      "/docs/synthetics"
+    ]
+  },
+  {
+    "url": "/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring",
+    "paths": [
+      "/docs/infrastructure"
```
Comment (rhetoric101) [inline src/data/taxonomy-redirects.json:2538]: @paperclypse, this 404s.

File: src/data/taxonomy-redirects.json [hunk 9]
```diff
@@ -2501,5 +2519,77 @@
...
+    ]
+  },
+  {
+    "url": "/docs/logs/get-started/get-started-log-management",
+    "paths": [
+      "/docs/logs"
```
Comment (rhetoric101) [inline src/data/taxonomy-redirects.json:2544]: @paperclypse, this 404s, too, so I'm guessing that my local system can't resolve any of these new entries.

**Review by rhetoric101 — APPROVED** [review]
@paperclypse, you deserve an award for touching all of these files! I inserted some comments below.

**Comment by roadlittledawn** [comment]
please hold off on this until dev en can commit to this branch. do not merge to develop

**Comment by roadlittledawn** [comment]
Closing this for now since this branch is not ready for a merge to develop until dev en can add work to it

File: src/data/taxonomy-redirects.json (hunk 4 above)
Comment (paperclypse) [reply src/data/taxonomy-redirects.json:611]: Because there isn't a good generic "agents" doc page.

**Pull request closed without merging**